woocommerce/woocommerce
# [CYS on Core] Remove the Feature Flag from the Customize Your Store flow on Core

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Remove the Feature flag for the Customize Your Store flow:

|  Before | After  |
|---|---|
| <img width="1996" alt="Screenshot 2024-02-21 at 10 44 17" src="https://github.com/woocommerce/woocommerce/assets/15730971/f466008d-af84-4315-ac31-415bb84c64e8"> | <img width="1995" alt="Screenshot 2024-02-21 at 10 45 40" src="https://github.com/woocommerce/woocommerce/assets/15730971/38a91401-0998-4475-8f06-9f4af86dd2ab"> |


Closes https://github.com/woocommerce/woocommerce/issues/44823



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Disable the **WooCommerce Beta Tester** plugin if you have it enabled
2. Access your Dashboard and head over to **WooCommerce > Home**
3. Make sure you can see the Customize your Store task listed, as exemplified in the screenshot below:

<img width="1995" alt="Screenshot 2024-02-21 at 10 39 29" src="https://github.com/woocommerce/woocommerce/assets/15730971/a17ba040-f5c9-41f3-9c58-99d3ed35735c">




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

Enable the Customize Your Store feature on Core.

#### Comment 

</details>
